data/json/items/gun/nail.json

```diff
@@ -1,5 +1,29 @@
 [
   {
+    "id": "coilgun",
+    "copy-from": "gun_base",
+    "type": "GUN",
+    "name": "coilgun",
+    "description": "A homemade gun, using electromagnets to accelerate a ferromagnetic projectile to high velocity.  Powered by UPS.",
+    "weight": 3341,
+    "volume": 11,
+    "price": 75000, "//": "Hard to make, plentiful and cheap ammo, and silent - people will want this thing.",
+    "bashing": 10,
+    "to_hit": -2,
+    "material": "steel",
+    "skill": "rifle",
+    "ammo": "nail",
+    "ranged_damage": 8,
```

Comment:
Needs much better stats to be relevant versus the nail rifle considering it's difficult to craft, requires UPS charges and is semi-auto only. Presumably the nails travel much faster and therefore have greater range and accuracy.

Possibly has a niche against armored opponents:
`range: 12` (improved due to velocity)
`pierce: 20` (improved due to velocity)
`damage: 4` (same as nail rifle)
`sight_dispersion: 60` (same as standard rifles)